reconcile/cli.py

```diff
@@ -203,9 +207,44 @@ def f(function):
     return f
 
 
-def run_integration(func, *args):
+def get_feature_toggle_default(feature_name: str, context: dict) -> bool:
+    return True
+
+
+@defer
+def get_feature_toggle_state(integration_name, defer=None):
+    api_url = os.environ.get('UNLEASH_API_URL')
+    client_access_token = os.environ.get('UNLEASH_CLIENT_ACCESS_TOKEN')
+    if not (api_url and client_access_token):
+        return True
+
+    # hide INFO logging from UnleashClient
+    logger = logging.getLogger()
+    default_logging = logger.level
+    logger.setLevel(logging.ERROR)
+    defer(lambda: logger.setLevel(default_logging))
```

Comment:
we have to start using different logging streams... but that's not only here, just something to keep in mind.